src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -50,7 +50,7 @@ public static function escapeTrailingBackslash($text)
         if ('\\' === substr($text, -1)) {
             $len = strlen($text);
             $text = rtrim($text, '\\');
-            $text .= str_repeat('<<', $len - strlen($text));
+            $text .= str_repeat('\0', $len - strlen($text));
```

Comment:
so, this relies on the fact that \0 is not printable, to use it as an escape character?
to be safe, what about removing `\0` from $text also?